inputs/paritytech/zombienet/flake-module.nix

```diff
@@ -49,6 +49,12 @@
           runtimeInputs = [ pkgs.nodejs paritytech-zombienet ] ++ runtimeDeps;
           text = ''
             export DEBUG="zombie*"
+            if [[ -d /tmp ]];
+            then 
+              echo "using /tmp"
+            else
+              mkdir --parents /tmp && chown 777 /tmp
```

Comment:
imho, we can remove `if` because `mkdir --parents /tmp && chown 777 /tmp` can be multi-time applied to exists directory without any errors if we have access to write

Reply:
i do not want to change host. the case only for hosts (like nixos containr) which do not have tmp. modifying user machine like that imho is bad.

Reply:
but adding tmp if there is none not so bad